### DISCUSSION TITLE
Is the project discontinued?

### DISCUSSION THREAD

**Discussion by Xisrr1** [anddea/revanced-patches#811] [Q&A]
I noticed that Francesco, one of the contributors, has recently left or took a break from the project. Could you please confirm if the project is still actively maintained or if there are any plans for its future development?

Thank you for your time and for the work you've done on this project.

**Comment by anddea** [comment]
The project has not been discontinued yet, although people wouldn't lose much if it were discontinued.

**Reply by Turhvjbufv** [reply]
Nah dude we will lose quite a bit, you are good.

**Comment by Xisrr1** [comment]
> The project has not been discontinued yet, although people wouldn't lose much if it were discontinued. 

People would lose Visual preferences icons in YT music, so it's not worth stopping. 😔

**Reply by vippium** [reply]
Not only this, custom snackbar too. My one of the fav feature. This is the reason for being with anddea patches, and not shifting to inotia's patches.

**Reply by vippium** [reply]
Just use it till it lasts. Anddea will update it less frequently. And also we have Francesco146 for the repo. When they have time, they will do it. Maybe once in a month.
